### ISSUE TITLE
Diseñar los experimentos (qué evaluar? si extracto con la respuesta aparece en la salida de ES? F1? Exact Match? Con qué dataset? cuántas preguntas hay para evaluar?) @aialarosa

### ISSUE THREAD

**Issue by aialarosa** [pln-fing-udelar/covid19-qa#17]
* Usar este conjunto de preguntas: 
https://docs.google.com/spreadsheets/d/16JEUMr9mKcKIsL8CH-52cbTRb_5OQyOsh_sgIcglT5w/edit#gid=0

* Evaluar diferentes configuraciones de ES: 
** con/sin diccionario
** cantidad de documentos a pasarle a beto: 10, 20, 30
** efecto de la flexibilización de la consulta (entre las preguntas ya hay algunas variantes que van a permitir evaluar esto)

* Revisar el log para ver si el fragmento que ya tenemos identificado es recuperado por ES.

* No calcular exact match, evaluar a mano las salidas, porque ahora hay muchos documentos más y la respuesta correcta puede aparecer en diferentes textos.

**Comment by bryant1410** [comment]
Demás.

> No calcular exact match, evaluar a mano las salidas, porque ahora hay muchos documentos más y la respuesta correcta puede aparecer en diferentes textos.

¿Pero exact match de strings o F1 no pueden capturar esto? Por ejemplo, si la respuesta aparece en un documento nuevo y dice "en Abril", cuando la correcta era "Abril" en otro doc.

**Comment by aialarosa** [comment]
En ese caso sí, pero en otros casos la respuesta puede estar escrita con
palabras completamente diferentes en un documento que no es el que usamos
para anotar respuestas. La vamos a considerar incorrecta cuando puede ser
correcta.

**Comment by bryant1410** [comment]
¿Qué tal si medimos el maximo exact match para las respuestas que mostramos en el sitio (hasta un máximo de 10)? El documento verdadero debería aparecer ahí, y creo que deberían ser pocos los casos en los que no y que además hay una buena respuesta en otro doc.

Digo para intentar automatizar la evaluación.

**Comment by aialarosa** [comment]
Sí, puede ser, es cierto que difícilmente la respuesta que tenemos anotada
no aparezca entre las correctas.
Igual no estaría mal detectar cualquier otra respuesta correcta, si es que
hay alguna.
